autopush-common/src/db/dynamodb/retry.rs

```diff
@@ -48,5 +50,5 @@ pub fn retryable_describe_table_error(
 
 /// Build an exponential retry policy
 pub fn retry_policy() -> RetryPolicy {
-    RetryPolicy::default().with_jitter(true)
+    RetryPolicy::exponential(Duration::from_millis(100))
```

Comment:
Can we revert this for now? If we're going to change it it really needs its own tracking issue/paper trail.